lib/dor/models/permissable.rb

```diff
@@ -0,0 +1,160 @@
+module Dor
```

Comment:
Permissable implements https://consul.stanford.edu/display/chimera/Repository+Roles+and+Permissions